learningresources/views.py

```diff
@@ -0,0 +1,39 @@
+"""
+Views for learningresources app.
+"""
+
+from django.shortcuts import render, redirect
+from django.core.urlresolvers import reverse
+from django.contrib.auth.decorators import login_required
+
+from learningresources.api import get_repos
+from learningresources.forms import RepositoryForm
+
+
+def welcome(request):
+    """
+    Greet the user, show available repositories.
+    """
+    return render(
+        request,
+        "welcome.html",
+        {"repos": get_repos(request.user)}
```

Comment:
I think you want to protect this with `@login_required` unless we want to allow for repos owned by anonymous as you are using the request.user here and that can be the stubby anonymous user object.